docs/versioned-plugins/inputs/log4j-v3.0.6.asciidoc

```diff
@@ -18,8 +18,6 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 include::{include_path}/plugin_header.asciidoc[]
 
-==== Description
```

Comment:
Removed the blank description section because it results in errors (this matches the fix used in other versions of the plugin).